react/Viewer/index.jsx

```diff
@@ -179,6 +186,10 @@ Viewer.propTypes = {
     showPanel: PropTypes.func,
     /** Content to be shown  */
     PanelContent: PropTypes.func
+  }),
+  /** File actions on mobile (to share or download the file, for example) */
+  mobileActionsProps: PropTypes.shape({
+    MobileActionsContent: PropTypes.func
```

Comment:
```suggestion
    MobileActionsContent: PropTypes.elementType
```
?

https://fr.reactjs.org/docs/typechecking-with-proptypes.html

>   // Un type d’élément React (ex. MyComponent).
  optionalElementType: PropTypes.elementType,